src/services/apis/EventsApi.js

```diff
@@ -71,6 +71,24 @@ export const api = {
             })
         })
         return Promise.all(promises)
+    },
+
+    getAllEvents: function(){
+        return fetch(basicApiLink + '/classes/events/all', {
+            method: "GET",
+            headers: {
+                "Content-Type": 'application/json'
+            }
+        })
+    },
+
+    getAllCoursesInEvents: function(name){
+        return fetch(basicApiLink + `/classes/where?event=${name}`, {
```

Comment:
In could be helpful to have the method for any params on where endpoint and as params it get object with all query params. Because of that body of this function would looks like this:
```javascript
return getCoursesFromWhereEndpoint({"event": name});
```